backend/i18n/i18n.go

```diff
@@ -99,11 +99,15 @@ func Init() {
 	_, _ = bundle.LoadMessageFileFS(fs, "lang/en.yaml")
 	_, _ = bundle.LoadMessageFileFS(fs, "lang/zh-Hant.yaml")
 	_, _ = bundle.LoadMessageFileFS(fs, "lang/fa.yaml")
+	_, _ = bundle.LoadMessageFileFS(fs, "lang/pt.yaml")
 }
 
 func UseI18nForCmd(lang string) {
 	if lang == "" {
-		lang = "zh"
+		lang = "en"
+	}
+	if lang == "pt-BR" {
+		lang = "pt"
 	}
 	if bundle == nil {
 		Init()
```

Comment:
There is no need to modify this code. The given function works as expected without any errors or unnecessary operations.

However, for the sake of completeness, you might want to:

1. Add error handling for loading the YAML files.
2. Return non-zero values instead of panicking.

The current implementation has no issues according to my review but it can be improved in terms of robustness and usability with added functionality like error responses and better return value management. 

In summary: keep it!